code/modules/reagents/chemistry/recipes/food.dm

```diff
@@ -102,6 +102,22 @@
 	for(var/i = 1, i <= created_volume, i++)
 		new /obj/item/reagent_containers/food/snacks/dough(location)
 
+///Cookies by Ume
+
+/datum/chemical_reaction/cookiedough
+	name = "Dough"
+	id = "dough"
+	result = null
+	required_reagents = list("milk" = 10, "flour" = 10, "sugar" = 5)
+	result_amount = 1
+	mix_message = "The ingredients form a dough. It smells sweet and yummy."
+
+/datum/chemical_reaction/cookiedough/on_reaction(datum/reagents/holder, created_volume)
+	var/location = get_turf(holder.my_atom)
+	for(var/i = 1, i <= created_volume, i++)
```

Comment:
Should do `for(var/i in 1 to created_volume)` here.

Reply:
I'm not sure why or what does that do, but it works as it is. It's the same code as the normal dough.

Reply:
It works the same way, but the way I've put is how we generally format for loops in paradise. I think it may also be faster?